### PR TITLE
Fix PDF export

### PR DESCRIPTION
**Pull request by MBausson**

### CODE REVIEW

**Comment by MBausson** [comment]
<del>
### Known issues

- Nested var
```css
body {
    --primary-color: #FF0000;
    --secondary-color: var(--primary-color);
}

.text-bg-danger {
    color: var(--secondary-color);
}
```

In this scenario, the `.text-bg-danger` color equals  `var(--primary-color)`, instead of `#FF0000`.

- Variables not defined/initialized (ex: `--bs-body-text-align`, `--bs-nav-link-font-size`). [See](https://github.com/twbs/bootstrap/issues/38474)

- Some variables are re-defined multiple times. For now, we just use the first occurence of the variable value

**Comment by MBausson** [comment]
https://stackoverflow.com/questions/70266865/how-can-i-direct-html2pdf-which-stylesheet-to-use-without-affecting-the-page

**Comment by MBausson** [comment]
In order to modify the style of the PDF result, we should apply styles on this class: `.html2pdf__container`

**Comment by B-Rass** [comment]
Update export button for move pages

File: app/views/frames/print.html.erb [hunk 1]
```diff
@@ -11,5 +11,7 @@
 %>
 
 <div class="container-fluid">
-  <%= render partial: 'servers/room', formats: :html, locals: { room: @room, islets: @servers_per_frames, pdf: true, view_side: (params[:view].present? ? params[:view] : 'back') } %>
+  <%= render partial: "servers/room",
+             formats: :html,
```
Comment (nicolas-brousse) [inline app/views/frames/print.html.erb]: Not necessary I guess

**Review by nicolas-brousse — APPROVED** [review]

**Pull request merged**